client/my-sites/email/titan-set-up-thank-you/style.scss

```
@@ -6,7 +6,8 @@
 }
 
 .thank-you__container {
```

I think this is the wrong approach -- I think the root problem is that this CSS selector is currently too general, so it can override the core specifications for this selector. Given that this CSS should only be used when we're within the emails section, can we limit this CSS to `.is-section-email`? I think we should still check that other uses of the Thank You component continue to work, however, as this overly general CSS might have been interfering in other contexts before this.
```suggestion
.is-section-email .thank-you__container {
```

---

In this case, I wonder if we should also remove `.thank-you__container` class, as we can completely distinguish this HTML component already only by `.is-section-email`

---

Plus I think, other instances of the `thank-you` components are not going to be affected, because this is the `titan-set-up-thank-you` component 🙂, which crates `thank-you` component with modified CSS.

---

And after rethinking a bit, it might be even better being a bit more descriptive about the class name, like `titan-set-up-thank-you__container` instead of `.is-section-email`, don't you think so?